cmd/kube-controller-manager/app/options/options.go

```diff
@@ -173,5 +174,10 @@ func (s *CMServer) AddFlags(fs *pflag.FlagSet) {
 	fs.DurationVar(&s.ControllerStartInterval.Duration, "controller-start-interval", s.ControllerStartInterval.Duration, "Interval between starting controller managers.")
 	fs.BoolVar(&s.EnableGarbageCollector, "enable-garbage-collector", s.EnableGarbageCollector, "Enables the generic garbage collector. MUST be synced with the corresponding flag of the kube-apiserver. WARNING: the generic garbage collector is an alpha feature.")
 	fs.Int32Var(&s.ConcurrentGCSyncs, "concurrent-gc-syncs", s.ConcurrentGCSyncs, "The number of garbage collector workers that are allowed to sync concurrently.")
+	fs.Float32Var(&s.NodeEvictionRate, "node-eviction-rate", 0.1, "Number of nodes per second on which pods are deleted in case of node failure when a zone is healthy.")
+	fs.Float32Var(&s.SecondaryNodeEvictionRate, "secondary-node-eviction-rate", 0.01, "Number of nodes per second on which pods are deleted in case of node failure when a zone is unhealty.")
+	fs.Int32Var(&s.LargeClusterSizeThreshold, "large-cluster-size-threshold", 20, "Number of nodes from which NodeController treats the cluster as large for the eviction logic purposes. For clusters smaller than this threshold evictions stop when the cluster is unhealty.")
+	fs.Int32Var(&s.UnhealthyZoneThreshold, "unhealthy-zone-threshold", 33, "Percentage of Nodes in a zone which needs to be not Ready (minimum 3) for zone to be treated as unhealthy. ")
```

Comment:
Would it be better to make this be a fraction rather than percentage? (e.g. 0.33 instead of 33)